spotify_db/Query.sql

```diff
@@ -0,0 +1,169 @@
+-- Top popular tracks
+SELECT t.track_name, ar.artist_name, al.album_name, t.popularity
+FROM tracks t
+JOIN artists ar ON t.artist_id = ar.artist_id
+JOIN albums al ON t.album_id = al.album_id
+WHERE t.popularity > (SELECT AVG(popularity) FROM tracks)
+ORDER BY t.popularity DESC; 
+
+-- Artists 
+SELECT 
+    ar.artist_name,
+    ar.followers,
+    COUNT(DISTINCT t.track_id) AS track_count,
+    COUNT(DISTINCT t.album_id) AS album_count,
+    COUNT(DISTINCT pt.playlist_id) AS playlist_count,
```

Comment:
Why do you need to `COUNT(DISTINCT)`? Are there any duplications after you join these tables?

Reply:
You can double check the data, and make a call whether you need `COUNT(DISTINCT)`, or `COUNT` should be sufficient enough.